databases/week3/week3-homework.sql

```diff
@@ -0,0 +1,226 @@
+-- https://github.com/HackYourFuture-CPH/databases/blob/main/lesson3/README.md#homework
+
+
+CREATE DATABASE MealSharingDB;
+USE MealSharingDB;
+
+
+-- Create all the sql for creating this data model: https://dbdiagram.io/d/5f0460690425da461f045a29
+
+
+CREATE TABLE Meal (
+    id INT PRIMARY KEY,
```

Comment:
If your id is a number that will increase by one, use AUTO_INCREMENT. Not only you don't have to specify the id when you create a new record, but also prevent common errors (two different users trying to create the same id at the same time). This goes for the three tables.